modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/build.gradle.mustache

```diff
@@ -104,8 +104,8 @@ if(hasProperty('target') && target == 'android') {
 dependencies {
     implementation 'io.swagger:swagger-annotations:1.5.24'
     implementation "com.google.code.findbugs:jsr305:3.0.2"
-    implementation 'com.squareup.okhttp3:okhttp:3.14.7'
-    implementation 'com.squareup.okhttp3:logging-interceptor:3.14.7'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.1'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.9.1'
```

Comment:
As part of this PR, can you please also update `build.sbt.mustache` by replacing `3.14.7` with `4.9.1` ?

Reply:
Ignore it. I will file another PR instead.